src/client/AccountsClient.js

```diff
@@ -48,13 +53,75 @@ export class AccountsClient {
       middleware,
     });
   }
-
   getState(): Map<string, any> {
     return this.store.getState().get('accounts');
   }
   user(): UserObjectType {
     return this.getState().get('user').toJS();
   }
+  tokens(): TokensType {
+    return {
+      accessToken: localStorage.getItem(ACCESS_TOKEN),
+      refreshToken: localStorage.getItem(REFRESH_TOKEN),
+    };
+  }
+  clearTokens() {
+    localStorage.removeItem(ACCESS_TOKEN);
+    localStorage.removeItem(REFRESH_TOKEN);
+  }
+  clearUser() {
+    this.store.dispatch(clearUser());
+  }
+  async resumeSession(): Promise<void> {
+    const { accessToken } = this.tokens();
+    if (accessToken) {
+      try {
+        const decodedAccessToken = jwtDecode(accessToken);
```

Comment:
The access token expires by itself over time. The access and refresh tokens are created and stored server side.